src/viem/saga.ts

```diff
@@ -146,30 +218,45 @@ function* getTransferSimulateContract({
   wallet: ViemWallet
   recipientAddress: string
   amount: BigNumber
-  tokenAddress: string
+  tokenId: string
   comment: string
   feeInfo: FeeInfo
 }) {
+  const tokenInfo = yield* call(getTokenInfo, tokenId)
   if (!wallet.account) {
     // this should never happen
     throw new Error('no account found in the wallet')
   }
+  const tokenAddress = tokenInfo?.address
+  if (!tokenAddress) {
+    throw new Error('token is native, no contract to simulate')
+  }
 
-  const tokenInfo = yield* call(getTokenInfo, tokenAddress)
-
-  const convertedAmount = BigInt(yield* call(tokenAmountInSmallestUnit, amount, tokenAddress))
+  const convertedAmount = BigInt(yield* call(tokenAmountInSmallestUnit, amount, tokenInfo?.tokenId))
 
   const encryptedComment = isStablecoin(tokenInfo)
     ? yield* call(encryptComment, comment, recipientAddress, wallet.account.address, true)
     : undefined
 
-  const feeFields = yield* call(getSendTxFeeDetails, {
-    recipientAddress,
-    amount,
-    tokenAddress,
-    feeInfo,
-    encryptedComment: encryptedComment || '',
-  })
+  const network = getNetworkFromNetworkId(tokenInfo?.networkId)
+  if (!network) {
+    throw new Error('invalid network for transfer')
+  }
+
+  // TODO (ACT-922): Use real fee estimation for Ethereum
```

Comment:
Since we don't yet have fee estimation for Ethereum, I'm punting on it for this ticket, and allowing non-Celo sends to work without fees being available. This is kind of a hack, and will need to be reverted once ACT-922 is complete.